plugins/mysql/mysqlembed.cpp

```diff
@@ -963,27 +963,19 @@ class MySQLRowStream : public CInterfaceOf<IRowStream>
 
 static __thread ThreadTermFunc threadHookChain;
 static __thread MySQLConnection *cachedConnection = NULL;
-static __thread const char *cachedServer = NULL;
-static __thread const char *cachedUser = NULL;
-static __thread const char *cachedPassword = NULL;
-static __thread const char *cachedDatabase = NULL;
-static __thread unsigned cachedPort = 0;
+static __thread const char *cachedOptions = NULL;
```

Comment:
not entirely up-to-date on the c11 thread_local (and maybe out of scope for this bug), but there's some push to use replace __thread:
http://stackoverflow.com/questions/13106049/c11-gcc-4-8-thread-local-performance-penalty